coderd/database/queries/insights.sql

```diff
@@ -113,23 +113,23 @@ SELECT
 FROM app_stats_by_user_and_agent
 GROUP BY access_method, slug_or_port, display_name, icon, is_app;
 
--- name: GetTemplateDailyInsights :many
--- GetTemplateDailyInsights returns all daily intervals between start and end
--- time, if end time is a partial day, it will be included in the results and
--- that interval will be less than 24 hours. If there is no data for a selected
+-- name: GetTemplateInsightsByInterval :many
+-- GetTemplateInsightsByInterval returns all intervals between start and end
+-- time, if end time is a partial interval, it will be included in the results and
+-- that interval will be shorter than a full one. If there is no data for a selected
```

Comment:
Nit: This function is named confusingly -- it looks like you can pass an arbitrary interval, but it only accepts an integer number of days. I believe you are addressing this in the follow-up PR?

Reply:
Correct, in the follow-up I will use it to specify a week (`IntervalDays=7`). I'm happy to rephrase the function name, but I have no good idea, `GetTemplateInsightsByIntervalDays`?

Reply:
No, that makes sense to me.